components/voice_registry_system.md

```diff
@@ -10,11 +10,25 @@ VRS's goal is to provide and promote voice openness and transparency.
 See [OVN vocabularies](https://github.com/open-voice-network/docs/blob/master/vocabulary.md)
 
 ### 1.0.3. Problem Context
-Today, voice assistance is dominated by proprietary, cloud-centric conversation platforms that deliver services using closed implementations.  Examples of these are Amazon Alexa, Google Assistant, Microsoft Cortana, and Apple Siri.  Every user invocation is received and managed by the hosting platform, which in turn will direct the user to one or more platform-specific assistance skills/actions/capsules.  Whenever an "explicit invocation" is invoked by the user -- that is, the user requests a specific destination -- each proprietary platform has a different method  (i.e., brand vocabulary, etc) and process for verification and connection to the user's desired entity.  
+The task of the Open Voice Network Voice Registry System (VRS) Work Group is to develop standards-based solutions for how _verbalized, explicitly-requested conversational agent destinations_ will,by another conversational agent, **be uniquely identified and connected.**
+
+This issue anticipates the continued growth of voice assistant systems that are independent of major platform technologies, and the emergence of a hybrid voice assistant ecosystem consisting of both general-purpose platform-based conversational agents and independent systems, most often managed and operated by enterprises. 
+
+The VRS will provide value to voice assistant system providers and operators, and to consumers. It will provide
```

Comment:
who are the operators? Is this define somewhere?

Reply:
The VRS will provide value to voice assistant system providers, organizations, content creators, and consumers. 

>> Discussed 06.10